examples/moonshot-mainseries/01-simulate-receptors.py

```diff
@@ -0,0 +1,228 @@
+
+Usage:
+  simulate.py --receptor=FILE --ligand=FILE --nsteps=INT [--selection=SELECT] [--initial=FILE] [--minimized=FILE] [--final=FILE] [--xtctraj=FILE] [--dcdtraj=FILE] [--pdbtraj=FILE] 
+  simulate.py (-h | --help)
+
+Options:
+  -h --help           Show this screen.
+  --receptor=FILE     Receptor PDB filename.
+  --ligand=FILE       Ligand SDF filename.
+  --nsteps=INT        Number of steps to run.
+  --selection=SELECT  MDTraj selection to use (e.g. 'not water') [default: all].
+  --initial=FILE      Write initial complex PDB file.  
+  --minimized=FILE    Write minimized complex PDB file.
+  --final=FILE        Write final complex PDB file.
+  --xtctraj=FILE      Generate XTC trajectory file.
+  --dcdtraj=FILE      Generate DCD trajectory file.
+  --pdbfile=FILE      Generate PDB trajectory file.
+
+"""
+from docopt import docopt
+arguments = docopt(__doc__, version='simulate 0.1')
+
+num_steps = int(arguments['--nsteps']) # number of integrator steps
+n_snapshots = int(num_steps / reporting_interval) # calculate number of snapshots that will be generated
+num_steps = n_snapshots * reporting_interval # recalculate number of steps to run
+
+log.info(f":gear:  Processing {arguments['--receptor']} and {arguments['--ligand']}")
+log.info(f':clock1:  Will run {num_steps*timestep / unit.nanoseconds:3f} ns of production simulation to generate {n_snapshots} snapshots')
+
+# check whether we have a GPU platform and if so set the precision to mixed
+speed = 0
+from openmm import Platform
+for i in range(Platform.getNumPlatforms()):
+    p = Platform.getPlatform(i)
+    # print(p.getName(), p.getSpeed())
+    if p.getSpeed() > speed:
+        platform = p
+        speed = p.getSpeed()
+
+if platform.getName() == 'CUDA' or platform.getName() == 'OpenCL':
+    platform.setPropertyDefaultValue('Precision', 'mixed')
+    log.info(f':dart:  Setting precision for platform {platform.getName()} to mixed')
+
+# Read the molfile into RDKit, add Hs and create an openforcefield Molecule object
+log.info(':pill:  Reading ligand')
+from rdkit import Chem
+rdkitmol = Chem.SDMolSupplier(arguments['--ligand'])[0]
+log.info(f':mage:  Adding hydrogens')
+rdkitmolh = Chem.AddHs(rdkitmol, addCoords=True);
+# ensure the chiral centers are all defined
+Chem.AssignAtomChiralTagsFromStructure(rdkitmolh);
+from openff.toolkit.topology import Molecule
+ligand_mol = Molecule(rdkitmolh);
+
+# Initialize a SystemGenerator
+log.info(':wrench:  Initializing SystemGenerator')
+from openmm import app
+from openmmforcefields.generators import SystemGenerator
+forcefield_kwargs = {'constraints': app.HBonds, 'rigidWater': True, 'removeCMMotion': False, 'hydrogenMass': 4*unit.amu }
+periodic_forcefield_kwargs = {'nonbondedMethod': app.PME}
+system_generator = SystemGenerator(
+    forcefields=['amber/ff14SB.xml', 'amber/tip3p_standard.xml'],
+    small_molecule_forcefield='openff-1.3.1',
```

Comment:
not `2.0.0`?

Reply:
I used `openff-1.3.1` here since (1) this was what we used for most of the Moonshot, and (2) `openff-2.0.0` has some known defects we would like to avoid until a bugfix release comes out.